peps/pep-0780.rst

```diff
@@ -0,0 +1,469 @@
+PEP: 780
+Title: ABI features as environment markers
+Author: Klaus Zimmermann <klaus_zimmermann@gmx.de>,
+        Ralf Gommers <ralf.gommers@gmail.com>
+Sponsor: Lysandros Nikolaou <lisandrosnik@gmail.com>
+Discussions-To: https://discuss.python.org/t/60007
```

Comment:
As previously suggested, leaving out Discussions-To for now (until the PR is merged, per the checklist)
```suggestion
```